frontend/src/assets/content/common/species/western-white-pine.md

```diff
@@ -0,0 +1,3 @@
+* Has 2 to 4 inch long needles.
+* Needles are long, blue-green, flexible and found in bundles of five
```

Comment:
@mtlaney is this line missing a period or is that deliberate? .. i noticed the others have punctuation

Reply:
bahhh, copy pasted from the issue!